torch/onnx/symbolic_helper.py

```diff
@@ -269,6 +269,35 @@ def _interpolate_size_to_scales(g, input, output_size, dim):
     return scales
 
 
+def _interpolate_get_scales_if_available(g, scales):
+    available_scales = _maybe_get_const(scales[0], 'f') != -1
+
+    if not available_scales:
+        return None
+
+    from torch.onnx.symbolic_opset9 import unsqueeze
```

Comment:
Seems weird that `symbolic_helper` is importing code from a specific symbolic opset.

Reply:
I agree that this is not ideal.
Do you have an idea of how to avoid doing that?

One way would be to have a helper function for unsqueeze in symbolic_helper (something like unsqueeze_helper()), that will call the right unsqueeze depending on the opset version (today the only one is the one in symbolic_opset9). 
Would that be better?

Reply:
I think it would be. Any common code can be pulled into helper, and symbolic can be downstream from it.